spec/system/task_spec.rb

```diff
@@ -1,36 +1,38 @@
 require 'rails_helper'
 
 RSpec.describe 'Task', type: :system do
+  
+  let(:project){ create(:project) }
+  let(:task){ create(:task, project: project) }
+  
   describe 'Task一覧' do
     context '正常系' do
       it '一覧ページにアクセスした場合、Taskが表示されること' do
-        # TODO: ローカル変数ではなく let を使用してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+        # Fixed TODO: ローカル変数ではなく let を使用してください
         visit project_tasks_path(project)
         expect(page).to have_content task.title
         expect(Task.count).to eq 1
         expect(current_path).to eq project_tasks_path(project)
       end
 
-      xit 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
-        # FIXME: テストが失敗するので修正してください
-        project = FactoryBot.create(:project)
-        task = FactoryBot.create(:task, project_id: project.id)
+      it 'Project詳細からTask一覧ページにアクセスした場合、Taskが表示されること' do
+        # Fixed FIXME: テストが失敗するので修正してください
         visit project_path(project)
         click_link 'View Todos'
-        expect(page).to have_content task.title
-        expect(Task.count).to eq 1
-        expect(current_path).to eq project_tasks_path(project)
+        #この表記で最後に指定したタブを指定する
```

Comment:
コメントはスペースを開けた方が見やすいですね。
```suggestion
        # この表記で最後に指定したタブを指定する
```

Reply:
8963e3f　コメントの後にスペースを追加しました。
また他にも同様のケースがありましたので修正しました。